core/storage/base_model/gae_models.py

```diff
@@ -441,27 +495,32 @@ class BaseHumanMaintainedModel(BaseModel):
     last_updated_by_human = (
         datastore_services.DateTimeProperty(indexed=True, required=True))
 
-    def put(self):
+    def put(self): # type: ignore[override]
```

Comment:
Any ignores that are not "ignore call" should have comments with explanation.

Reply:
Done